src/mono/browser/build/BrowserWasmApp.targets

```diff
@@ -609,4 +610,29 @@
 
     <Exec Condition="'$(OS)' != 'Windows_NT'" Command="chmod a+x &quot;$(WasmRunV8ScriptPath)&quot;" />
   </Target>
+
+  <!-- Copy dotnet.d.ts to wwwroot for TypeScript development experience -->
+  <Target Name="_CopyDotnetTypeScriptDefinitions" 
```

Comment:
This executes only for runtime tests.

https://github.com/dotnet/runtime/blob/main/src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets